### PR TITLE
tests: use the tfaga wrapper in the testing actions

### PR DESCRIPTION
**Pull request by zmiklank**
And [use better error logging in container tests action](https://github.com/sclorg/postgresql-container/commit/da95f53c294f80eb6a226ee5efbe69b7f241bbd6).

### CODE REVIEW

File: .github/workflows/container-tests.yml
Comment (phracek) [inline .github/workflows/container-tests.yml:44]: This part is missing in `.github/workflows/openshift-tests.yml`

**Review by phracek — CHANGES_REQUESTED** [review]
Please address my comment.

File: .github/workflows/container-tests.yml
Comment (zmiklank) [reply .github/workflows/container-tests.yml:44]: Yes, it was intended. The same is in python and varnish

Point is, that we want only one status line for distgen check. Not one from container tests and one from OCP tests.
Thus the status is being created only from container tests.

I realize, that this is not ideal. Do you have any better idea?

Comment (phracek) [reply .github/workflows/container-tests.yml:44]: What about if we schedule tests only for "OpenShift"? Then dist-gen check is skipped.

Comment (phracek) [reply .github/workflows/container-tests.yml:44]: I don't see any problem with duplication. distgen report should be checked always.

**Review by phracek — CHANGES_REQUESTED** [review]

File: .github/workflows/container-tests.yml
Comment (zmiklank) [reply .github/workflows/container-tests.yml:44]: But what if the content is not regenerated properly? Then we will be testing incorrect source codes.

Comment (zmiklank) [reply .github/workflows/container-tests.yml:44]: Oh, I see, there is a check in OCP tests too, but does not produce status.

Comment (zmiklank) [reply .github/workflows/container-tests.yml:44]: Please note that 'distgen check for OCP tests feature' got LGTM when it was implemented[1]. 
Also the distgen check is not connected to this PR.

If you want to change the functionality I would suggest discussing this on the container meeting and then maybe creating and issue or PR.


[1] https://github.com/sclorg/s2i-python-container/pull/558

Comment (phracek) [reply .github/workflows/container-tests.yml:44]: In case of dist-gen check failed, it would be nice to have a reason, but let's not block this pull request. It is a nice to have.

**Review by phracek — APPROVED** [review]

**Pull request merged**
